src/pages/careers/internships.js

```diff
@@ -9,19 +9,38 @@ import Footer from "../../sections/General/Footer";
 import InternshipPage from "../../sections/Careers/Careers-Internship-grid";
 
 import { GlobalStyle } from "../../sections/app.style";
-import theme from "../../theme/app/themeStyles";
+import darktheme from "../../theme/app/darkThemeStyles";
+import lighttheme from "../../theme/app/themeStyles";
+import { useCookies } from "react-cookie";
+import { useEffect } from "react";
+import { useState } from "react";
 
-const Internships = () => (
-  <ThemeProvider theme={theme}>
-    <Layout>
-      <GlobalStyle />
-      <SEO title="Open Source Internship Programs" description="The Service Mesh Community at Layer5 - Learn, Share, Engage.
+const Internships = () =>  {
+  const [cookies, setCookie] = useCookies(["user"]);
+  const [theme, setTheme] = useState(cookies.Theme);
+  const [themeToggle, setthemeToggle] = useState("");
+  const handle = () => {
+    theme === "light" ? setthemeToggle(false) : setthemeToggle(true);
+    setCookie("Theme", theme, { path: "/" });
```

Comment:
Can't this function be moved inside the `useEffect` hook itself?
Also, shouldn't the path be `/careers/internships`?

Reply:
> The approach is nice but would require updates in color in some places. Though I think we can avoid the use of cookies just to set the theme state, it seems like unnecessary overhead. Secondly, a lot of time people disable cookies in their browser, and in that case, the feature might not work as expected.

I actually preferred cookies because I saw in many websites which have dark themes always have used cookies to enable it, but Yes, if the browser disables cookies, Our Website will always start with the light theme as default without remembering its previous theme.